clusterloader2/pkg/test/interface.go

```diff
@@ -64,3 +66,14 @@ type Executor interface {
 	ExecutePhase(ctx Context, phase *api.Phase) *errors.ErrorList
 	ExecuteObject(ctx Context, object *api.Object, namespace string, replicaIndex int32, operation OperationType) *errors.ErrorList
 }
+
+// TestReporter is an interface for reporting tests results.
+type TestReporter interface {
+	SetTestID(name string)
+	GetTestID() string
+	GetNumberOfFailedSpecs() int
```

Comment:
TODO: Rename the method to sth like `GetNumberOfFailedTestItems()`.

Reply:
What about this TODO?

Reply:
Done.